### DISCUSSION TITLE
How to target multiple elements?

### DISCUSSION THREAD

**Discussion by ghost** [bigskysoftware/htmx#893] [Q&A] [answered]
For example, I have html like below:
```
<div>
  <div hx-post="example/url" target="this">...</div>

  <div class="modal">
    <div hx-post="example/url" target="this">...</div>
  </div>
</div>
```
I want to replace both div whenever I clicked one of them, how should I do it?

**Comment by David-Guillot** [comment]
Hi there,

Seems suboptimal to me to load the same HTML fragment into multiple divs of your page, but if you're sure this is what you want, you'll have to write some custom JS, as AFAIK htmx does not provide any native way to achieve it.

The easiest way I can think of would be to write your own htmx extension that would expose a new hx-attribute which would be a CSS selector of additional target elements. This way you would be able to call htmx internal API to swap the loaded content into wherever you want.

I hope I'm being clear and that helped 😉

**Comment by David-Guillot** [comment, marked as answer]
Oh wait, I might have an easier solution: if you mark the root element of your AJAX response as [hx-swap-oob](https://htmx.org/attributes/hx-swap-oob/), you should be able to pass a CSS selector that matches both divs, and your AJAX response should be swapped twice.

Let me know if this works as expected, as it's way simpler than my first idea.

**Reply by ghost** [reply]
This is exactly what I want, thank you!
